eng/pipelines/preview-typespec.yml

```diff
@@ -0,0 +1,16 @@
+parameters:
+- name: RunUnitTests
+  displayName: Run unit tests
+  type: boolean
+  default: true
+- name: RunGenerationChecks
+  displayName: Run code generation checks
+  type: boolean
+  default: true
+
+extends:
+- template: /eng/pipelines/preview.yml
+  parameters:
+    UseTypeSpecNext: true
```

Comment:
i wonder how this works - this uses the next version of typespec and overrides the version used in TCGC?

Reply:
You should review the latest commit.

But yes to your question, it will override all the versions in our package.json.